cmd/agent/flags.go

```diff
@@ -80,7 +80,7 @@ var flags = []cli.Flag{
 	&cli.IntFlag{
 		EnvVars: []string{"WOODPECKER_MAX_PROCS"},
 		Name:    "max-procs",
-		Usage:   "agent parallel builds",
+		Usage:   "agent parallel pipelines",
```

Comment:
for sure not "stage" ?

Reply:
Yip that should be "stage" or whatever we want to call it.

Reply:
at this point I think we should call it that way :sweat:  ... - I dont like how it sounds but well that's not a technical argument and so it does not count

Reply:
uh we use a "workspace" in each "stage" - so why not "workflow"

Reply:
https://github.com/woodpecker-ci/woodpecker/pull/1224#pullrequestreview-1142877194